titan-test/src/main/java/com/thinkaurelius/titan/graphdb/TitanGraphTest.java

```diff
@@ -2518,8 +2518,12 @@ private void awaitAllThreadsReady() {
      ==================================================================================*/
 
     @Test
-    @SuppressWarnings("deprecation")
     public void testVertexCentricQuery() {
+        int noVertices = 10000;
```

Comment:
is this what you wanted to do here? this is different from the previous PR that called `testVertexCentricQuery(10000)`